src/Shared/HttpSys/NativeInterop/HttpSysRequestHeader.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.AspNetCore.HttpSys.Internal;
 
-internal enum HttpSysRequestHeader
+internal enum HttpSysRequestHeader : byte
```

Comment:
With the `byte` a `ROS<HttpSysRequestHeader>` can be created that referes to assembly's static data segment.

Reply:
```suggestion
// With the byte a ROS<HttpSysRequestHeader> can be created that referes to assembly's static data segment.
internal enum HttpSysRequestHeader : byte
```

Reply:
That's the obvious thing 😃
Thanks!